extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/export/handlers/PrometheusHandler.java

```diff
@@ -34,22 +34,25 @@ public void handle(RoutingContext routingContext) {
                     .setStatusMessage("Unable to resolve Prometheus registry instance");
         } else {
             ManagedContext requestContext = Arc.container().requestContext();
+            var acceptHeader = routingContext.request().getHeader("Accept").equals("text/plain") ? TextFormat.CONTENT_TYPE_004
```

Comment:
`routingContext.request().getHeader("Accept")` can return `null`

Reply:
Corrected the point. Could you please alow the build to pass again ?